vignettes/readr.Rmd

```diff
@@ -253,12 +222,12 @@ Use the `col_types` argument to override the default choices. There are two ways
     
     ```r
       read_csv("iris.csv", col_types = list(
-      Sepal.Length = "d",
-      Sepal.Width = "d",
-      Petal.Length = "d",
-      Petal.Width = "d",
-      Species = col_factor(c("setosa", "versicolor", "virginica"))
-    ))
+        Sepal.Length = "d",
```

Comment:
Let's convert this to a normal chunk, with `eval = FALSE`

In general, this is good for keeping formatting and styling consistent.